### PR TITLE
Update pktmon-wac-extension.md

### PR DESCRIPTION
**Pull request by itamar56**
There is a mistake in the server version, the document says: "To use the tool, the target server needs to be running Windows Server 2019 version 1903 (19H1) and above." This version doesn't exist. It should be Windows Server, Version 1903

### CODE REVIEW

**Comment by PRMerger3** [comment]
@itamar56 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by itamar56** [comment]
Hi team, Kind reminder.
@khdownie

File: WindowsServerDocs/networking/technologies/pktmon/pktmon-wac-extension.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Packet Monitor (Pktmon) is an in-box, cross-component network diagnostics tool f
 Windows Admin Center is a locally-deployed, browser-based management tool that lets you manage your Windows Servers with no Azure or cloud dependency. Windows Admin Center gives you full control over all aspects of your server infrastructure and is particularly useful for managing servers on private networks that are not connected to the Internet. Windows Admin Center is the modern evolution of "in-box" management tools, like Server Manager and MMC.
 
 ## Before you start
-- To use the tool, the target server needs to be running Windows Server 2019 version 1903 (19H1) and above.
+- To use the tool, the target server needs to be running Windows Server, version 1903 (Semi-Anual Channel) and above.
```
Comment (khdownie) [inline WindowsServerDocs/networking/technologies/pktmon/pktmon-wac-extension.md:23]:
```suggestion
- To use the tool, the target server needs to be running Windows Server 2019, version 1903 or later.
```

**Review by khdownie — CHANGES_REQUESTED** [review]

**Review by khdownie — APPROVED** [review]

**Comment by khdownie** [comment]
#sign-off

**Pull request merged**